quantumdb-core/src/main/java/io/quantumdb/core/backends/Config.java

```diff
@@ -106,6 +109,7 @@ public Backend getBackend() {
 				}
 			}
 			catch (ReflectiveOperationException e) {
+				throw new IllegalArgumentException("Something went wrong selecting backends.");
```

Comment:
This does change the behaviour of this method. Instead of skipping backends that can't be loaded (and trying the next one), it now throws an error. I'm ok with the change, but at least pass on the exception so it's logged to the user:

```suggestion
				throw new IllegalArgumentException("Something went wrong selecting backends.", e);
```